tests/evm_backends/pyevm_env.py

```diff
@@ -4,16 +4,16 @@
 
 import rlp
 from cached_property import cached_property
-from eth.abc import ChainAPI, ComputationAPI
+from eth.abc import ChainAPI, ComputationAPI, VirtualMachineAPI
```

Comment:
this is a nit but i'm not a huge fan of importing things that are only used in type signatures, i am starting to prefer guarding these with `if TYPE_CHECKING`